source/administration/config-settings.rst

```diff
@@ -290,34 +290,89 @@ Set policy on who can invite others to a team using "Invite New Member" to invit
 | This feature's ``config.json`` setting is ``"RestrictTeamInvite": "all"`` with options ``all``, ``team_admin`` and ``system_admin`` for above settings respectively. |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-Enable public channel management permissions for
+Enable public channel creation for
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Restrict the permission levels required to create, delete, rename, and set the header or purpose for public channels. The last member of a public channel has the ability to delete the channel regardless of their permission level.
+Restrict the permission levels required to create public channels.
```

Comment:
Would this be "permission level"? (Since only one permission level is required to do the action)